From 30137148967bd1f918af493ad0fe82384924ccdf Mon Sep 17 00:00:00 2001
From: chrome-weblayer-builder
 <chrome-weblayer-builder@chops-service-accounts.iam.gserviceaccount.com>
Date: Fri, 10 Jun 2022 01:23:33 +0000
Subject: [PATCH 02/59] [lacros skew tests] Refresh skew tests for M104

This CL updates the ash version ['104.0.5111.0'] for Lacros version skew testing.
If this CL caused regressions, please revert and stop the autoroller
at https://luci-scheduler.appspot.com/jobs/chrome/lacros-version-skew-roller
Also please file a bug to OS>LaCrOS>Partner, and CC svenzheng@chromium.org,
yjt@google.com.

R=rubber-stamper@appspot.gserviceaccount.com

Bug: None
Change-Id: I95e56d3c445e495ed1d9a36441249615db9af009
Requires-Testing: True
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3696160
Bot-Commit: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Auto-Submit: chrome-weblayer-builder <chrome-weblayer-builder@chops-service-accounts.iam.gserviceaccount.com>
Commit-Queue: Rubber Stamper <rubber-stamper@appspot.gserviceaccount.com>
Cr-Commit-Position: refs/branch-heads/5005@{#1161}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 testing/buildbot/chromium.chromiumos.json | 20 +++---
 testing/buildbot/chromium.fyi.json        | 80 +++++++++++------------
 testing/buildbot/variants.pyl             |  8 +--
 3 files changed, 54 insertions(+), 54 deletions(-)

diff --git a/testing/buildbot/chromium.chromiumos.json b/testing/buildbot/chromium.chromiumos.json
index 82941bc7cd72..4ce3bb7c9e91 100644
--- a/testing/buildbot/chromium.chromiumos.json
+++ b/testing/buildbot/chromium.chromiumos.json
@@ -6018,21 +6018,21 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "isolate_profile_data": true,
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "dimension_sets": [
@@ -6044,7 +6044,7 @@
         },
         "test": "lacros_chrome_browsertests",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -6161,21 +6161,21 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "isolate_profile_data": true,
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "dimension_sets": [
@@ -6187,7 +6187,7 @@
         },
         "test": "lacros_chrome_browsertests_run_in_series",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests_run_in_series/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
diff --git a/testing/buildbot/chromium.fyi.json b/testing/buildbot/chromium.fyi.json
index 3d4f3c3b2c9f..f3e2e4997888 100644
--- a/testing/buildbot/chromium.fyi.json
+++ b/testing/buildbot/chromium.fyi.json
@@ -87176,28 +87176,28 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "isolate_profile_data": true,
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "service_account": "chromium-tester@chops-service-accounts.iam.gserviceaccount.com"
         },
         "test": "lacros_chrome_browsertests",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -87294,28 +87294,28 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "isolate_profile_data": true,
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "service_account": "chromium-tester@chops-service-accounts.iam.gserviceaccount.com"
         },
         "test": "lacros_chrome_browsertests_run_in_series",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests_run_in_series/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -88682,20 +88682,20 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "dimension_sets": [
@@ -88708,7 +88708,7 @@
         },
         "test": "lacros_chrome_browsertests",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -88825,20 +88825,20 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "dimension_sets": [
@@ -88851,7 +88851,7 @@
         },
         "test": "lacros_chrome_browsertests_run_in_series",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests_run_in_series/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -90381,20 +90381,20 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "dimension_sets": [
@@ -90407,7 +90407,7 @@
         },
         "test": "lacros_chrome_browsertests",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -90524,20 +90524,20 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "dimension_sets": [
@@ -90550,7 +90550,7 @@
         },
         "test": "lacros_chrome_browsertests_run_in_series",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests_run_in_series/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -91286,27 +91286,27 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "service_account": "chromium-tester@chops-service-accounts.iam.gserviceaccount.com"
         },
         "test": "lacros_chrome_browsertests",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       },
       {
         "args": [
@@ -91382,27 +91382,27 @@
       },
       {
         "args": [
-          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome"
+          "--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome"
         ],
         "merge": {
           "args": [],
           "script": "//testing/merge_scripts/standard_gtest_merge.py"
         },
-        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5109.0",
+        "name": "lacros_chrome_browsertests_run_in_series Lacros version skew testing ash 104.0.5111.0",
         "swarming": {
           "can_use_on_swarming_builders": true,
           "cipd_packages": [
             {
               "cipd_package": "chromium/testing/linux-ash-chromium/x86_64/ash.zip",
-              "location": "lacros_version_skew_tests_v104.0.5109.0",
-              "revision": "version:104.0.5109.0"
+              "location": "lacros_version_skew_tests_v104.0.5111.0",
+              "revision": "version:104.0.5111.0"
             }
           ],
           "service_account": "chromium-tester@chops-service-accounts.iam.gserviceaccount.com"
         },
         "test": "lacros_chrome_browsertests_run_in_series",
         "test_id_prefix": "ninja://chrome/test:lacros_chrome_browsertests_run_in_series/",
-        "variant_id": "Lacros version skew testing ash 104.0.5109.0"
+        "variant_id": "Lacros version skew testing ash 104.0.5111.0"
       }
     ]
   },
diff --git a/testing/buildbot/variants.pyl b/testing/buildbot/variants.pyl
index 389bb3539547..4f39dbcf07ba 100644
--- a/testing/buildbot/variants.pyl
+++ b/testing/buildbot/variants.pyl
@@ -28,15 +28,15 @@
   },
   'LACROS_VERSION_SKEW_CANARY': {
     'args': [
-      '--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5109.0/test_ash_chrome',
+      '--ash-chrome-path-override=../../lacros_version_skew_tests_v104.0.5111.0/test_ash_chrome',
     ],
-    'identifier': 'Lacros version skew testing ash 104.0.5109.0',
+    'identifier': 'Lacros version skew testing ash 104.0.5111.0',
     'swarming': {
       'cipd_packages': [
         {
           'cipd_package': 'chromium/testing/linux-ash-chromium/x86_64/ash.zip',
-          'location': 'lacros_version_skew_tests_v104.0.5109.0',
-          'revision': 'version:104.0.5109.0',
+          'location': 'lacros_version_skew_tests_v104.0.5111.0',
+          'revision': 'version:104.0.5111.0',
         },
       ],
     },
-- 
2.37.0

